recipe/meta.yaml

```diff
@@ -101,9 +101,35 @@ requirements:
     - packaging
     - pymongo
 test:
+  requires:
+  # This environment specification must be maintained in sync with the one upstream:
+  # See: https://github.com/man-group/ArcticDB/blob/master/environment_unix.yml
+  # See `options.extras_require`: https://github.com/man-group/ArcticDB/blob/master/setup.cfg
+    - pytest
+    - pyarrow
+    - pytest-cpp
+    - pytest-timeout
+    - pytest-xdist
+    - packaging
+    - future
+    - mock
+    - boto3
+    - moto
+    - flask  # Used by moto
+    - flask-cors
+    - hypothesis <6.73
+    - azure-storage-blob
+    - azure-identity
+    - coverage
+    - asv
+    - virtualenv
+    - pymongo
+  source_files:
+    - python/tests
   imports:
     - arcticdb
-
+  commands:
+    - pytest tests -n2
```

Comment:
Unless the working directory is specified somewhere else, I think this should be `python/tests`